From: Takashi Iwai <tiwai@suse.de>
Subject: Fix kABI for musb flush_irq_work field addition
Patch-mainline: Never, kABI fix for SLE15
References: bsc#1085536

The patch patches.fixes/0001-USB-musb-fix-late-external-abort-on-suspend.patch
introduces a new bit field flush_irq_work to struct musb, and this
breaks kABI.  Wrap it with __GENKSYMS__ for kABI compatibility.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/usb/musb/musb_core.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/usb/musb/musb_core.h
+++ b/drivers/usb/musb/musb_core.h
@@ -427,7 +427,9 @@ struct musb {
 	unsigned		test_mode:1;
 	unsigned		softconnect:1;
 
+#ifndef __GENKSYMS__
 	unsigned		flush_irq_work:1;
+#endif
 
 	u8			address;
 	u8			test_mode_nr;
